client/components/feature-modules/blocks/EXAMPLE_USAGE.tsx

```
@@ -0,0 +1,331 @@
+        } else {
+            console.log("Validation failed or error occurred");
+        }
+    };
+
+    const handleOpenDrawer = (containerBlockId: string) => {
+        openDrawer(containerBlockId);
+    };
+
+    return (
+        <div>
+            <button onClick={() => handleStartInlineEdit("block-1")}>Edit Block</button>
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)